pkgs/development/python-modules/plyer/default.nix

```diff
@@ -0,0 +1,64 @@
+{ lib, buildPythonPackage, fetchFromGitHub, fetchpatch, keyring, mock, pytestCheckHook, stdenv }:
+
+buildPythonPackage rec {
+  pname = "plyer";
+  version = "2.0.0";
+
+  src = fetchFromGitHub {
+    owner = "kivy";
+    repo = "plyer";
```

Comment:
```suggestion
    repo = pname;
```

Reply:
whoops, did I accidentally resolve this earlier? Fixed now